include/board.h

```diff
@@ -1,11 +1,50 @@
 #pragma once
+//
```

Comment:
please prefer doxygen format `///` when writing comments

Reply:
Changed to a doxygen-styled comment